src/modes/marquee.js

```diff
@@ -19,6 +19,11 @@ RectangularDraw.onSetup = function () {
   this.clearSelectedFeatures();
   doubleClickZoom.disable(this);
 
+  setTimeout(() => {
+    if (!this.map || !this.map.dragPan) return;
+    this.map.dragPan.disable();
+  }, 0);
```

Comment:
setTimeout "timeout" is 0 by default, btw

Reply:
ty!